test/e2e/feature/feature.go

```diff
@@ -432,6 +432,10 @@ var (
 	// TODO: remove when SELinuxMount feature gate is enabled by default.
 	SELinuxMountReadWriteOncePodOnly = framework.WithFeature(framework.ValidFeatures.Add("SELinuxMountReadWriteOncePodOnly"))
 
+	// SeparateDiskTest (SIG-node, used for testing separate image filesystem <https://kep.k8s.io/4191>)
```

Comment:
This test is not related to KEP 4191. 

This is a feature that has been in kubelet for a long time already. I wanted to add e2e tests to make sure everything works.